package.json

```diff
@@ -51,7 +51,7 @@
   "dependencies": {
     "custom-card-helpers": "^1.8.0",
     "lit": "^2.7.2",
-    "suncalc3": "link:suncalc3"
+    "suncalc3": "^2.0.5"
```

Comment:
What's the effect of this change?

Reply:
I couldn't install link:suncalc3 locally. I got an error: "npm ERR! Unsupported URL Type "link:": link:suncalc3"

So now install the latest version
https://www.npmjs.com/package/suncalc3